sample-app/src/main/res/layout/discount.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Does this need to be a `ConstraintLayout`? It only has a single child. Can the `TextView` just be the root? Or at least make the root a `FrameLayout` instead?